app/src/main/res/layout-land/fragment_presets.xml

```diff
@@ -85,6 +85,32 @@
         app:layout_constraintBottom_toTopOf="@+id/phrases_back_button"
         app:layout_constraintTop_toBottomOf="@+id/category_view" />
 
+    <TextView
+        android:id="@+id/empty_phrases_text"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/custom_category_empty"
+        style="@style/CustomCategoryEmptyText"
+        app:layout_constraintVertical_chainStyle="packed"
+        app:layout_constraintWidth_percent=".75"
+        app:layout_constraintTop_toTopOf="@id/phrases_view"
+        app:layout_constraintBottom_toTopOf="@id/empty_add_phrase_button"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"/>
+
+    <com.willowtree.vocable.customviews.KeyboardButton
+        android:id="@+id/empty_add_phrase_button"
+        android:layout_width="0dp"
+        android:layout_height="64dp"
```

Comment:
Don't forget to move these heights to a corresponding `dimens` file. It looks like they're all `64dp`, so you can just add it to the base `dimens` file.